docker-compose.yml

```diff
@@ -1,12 +1,11 @@
-version: "3"
-
 services:
   kutt:
+    platform: linux/amd64
```

Comment:
Added this as it currently seems to fail to install some dependencies on arm chips